src/metabase/cmd/endpoint_dox.clj

```diff
@@ -5,22 +5,120 @@
             [metabase.plugins.classloader :as classloader]
             [metabase.util :as u]))
 
+;;;; API docs intro
+
+(defn- api-docs-intro!
```

Comment:
```suggestion
(defn- api-docs-intro
```

`!` is for functions that have side effects such as writing files to disk or affecting global state